quinn-proto/src/crypto.rs

```diff
@@ -20,6 +20,7 @@ use webpki::DNSNameRef;
 
 use crate::coding::{BufExt, BufMutExt};
 use crate::packet::{ConnectionId, PacketNumber, LONG_HEADER_FORM};
+use crate::transport_error::Error as TransportError;
```

Comment:
More convenient to import the already-aliased version of this from the crate root.

Reply:
👍